xml/System.ServiceModel.Configuration/BasicHttpBindingElement.xml

```diff
@@ -168,8 +177,8 @@
         <ReturnType>System.ServiceModel.HostNameComparisonMode</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets or sets a value that indicates whether the hostname is used to reach the service when matching the URI.</summary>
+        <value>The <see cref="T:System.ServiceModel.HostnameComparisonMode" /> value that indicates whether the hostname is used to reach the service when matching on the URI. The default value is <see cref="F:System.ServiceModel.HostnameComparisonMode.StrongWildcard" />, which ignores the hostname in the match.</value>
```

Comment:
Two links are broken here; it should be Host**N**ameComparisonMode in both cases.